src/Compilers/CSharp/Portable/Operations/CSharpOperationFactory_Methods.cs

```diff
@@ -92,14 +92,20 @@ private IVariableDeclaratorOperation CreateVariableDeclarator(BoundLocal boundLo
             return new VariableDeclarator(boundLocal.LocalSymbol, initializer: null, semanticModel: _semanticModel, syntax: boundLocal.Syntax, type: null, constantValue: default, isImplicit: false);
         }
 
-        private IOperation CreateBoundCallInstanceOperation(BoundCall boundCall)
+        private IOperation CreateReceiverOperation(BoundNode instance, ISymbol symbol)
         {
-            if (boundCall.Method == null || boundCall.Method.IsStatic)
+            if (instance == null || instance.Kind == BoundKind.TypeExpression)
             {
                 return null;
             }
 
-            return Create(boundCall.ReceiverOpt);
+            // Static methods cannot have an implicit this receiver
+            if (symbol != null && symbol.IsStatic && instance.WasCompilerGenerated && instance.Kind == BoundKind.ThisReference)
```

Comment:
Typo: static members

Reply:
Fixed.

---
In reply to: [148815860](https://github.com/dotnet/roslyn/pull/22972#discussion_r148815860) [](ancestors = 148815860)